usrp3/tools/scripts/auto_inst_x310.yml

```diff
@@ -0,0 +1,31 @@
+# auto-inst file for X310
+# To regenerate rfnoc_ce_auto_inst_x310.v:
+#   ./uhd_image_builder.py  -d x310 -t X310_RFNOC_HG -m 10 --fill-with-fifos -y auto_inst_x310.yml -o rfnoc_ce_auto_inst_x310.v
+
+- block: ddc
+  parameters:
+    - name: NOC_ID
```

Comment:
Can these be key/value pairs instead of name/value entries? Something like
```yaml
parameters:
  NOC_ID: 64'hDDC0_0000_0000_0001
  NUM_CHAINS: 1
```

Reply:
Obv, that comes with further code changes.

Reply:
Ah! Great suggestion. I like it. Will change tomorrow